functions-methods.js

```diff
@@ -33,4 +81,31 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+//STAPPENPLAN
+// [X] 1     => Maak een functie genaamd checkEmailValidity met parameter [emailAddress]
+// [X] 2     => Laat de functie checken of het e-mailadres valide is
+// [X]  2.1  => Laat de functie controleren of '@' in het e-mailadres voorkomt (string methode includes())
+// [X]  2.2  => Laat de functie controleren of er géén ',' in het e-mailadres voorkomt (!= string methode includes()?)
+// [X]  2.3  => Laat de functie controleren of het laatste karakter niet een '.' is (!= string methode endsWith
+// [X] 3     => Console log uitkomst true of false afhankelijk van de criteria.
+// [X] 4     => Roep de functie meerdere malen aan met verschillende argumenten om hem te testen.
+
+//Deze functie checkt of een e-mailadres aan bepaalde voorwaarden voldoet (er komst een '@' in het adres voor, er komt geen ',' in voor en het adres eindigt niet met '.')
+function checkEmailValidity (emailAddress) {
+   if( (emailAddress.includes("@")) && (!(emailAddress.includes(","))) && (!(emailAddress.endsWith(".")))) {
+       return true
+    } else {
+       return false
+    }
```

Comment:
Je had de uitkomst van de expressie meteen terug kunnen geven:
```suggestion
   return emailAddress.includes("@") && !emailAddress.includes(",") && !emailAddress.endsWith(".");
```